src/main/java/com/redislabs/redisgraph/impl/Utils.java

```diff
@@ -56,6 +52,7 @@ private static String quoteString(String str){
      * @param args - query arguments
      * @return formatted query
      */
+    @Deprecated
```

Comment:
please add @deprecated to the JDoc pointing to the new method

Reply:
done